src/AggregateFunctions/AggregateFunctionSparkbar.cpp

```diff
@@ -54,10 +54,16 @@ AggregateFunctionPtr createAggregateFunctionSparkbar(const std::string & name, c
     if (params.size() == 3)
     {
         if (params.at(1).getType() != arguments[0]->getDefault().getType() || params.at(2).getType() != arguments[0]->getDefault().getType())
-        {
             throw Exception("The second and third parameters are not the same type as the first arguments for aggregate function " + name, ErrorCodes::ILLEGAL_TYPE_OF_ARGUMENT);
-        }
     }
+
+    /// This value represents the width of the resulting sparkbar.
```

Comment:
Kind of related, I noticed that the [sparkbar docs](https://clickhouse.com/docs/en/sql-reference/aggregate-functions/reference/sparkbar/) could be improved:
- `SETTINGS index_granularity = 8192` is an unnecessary distraction and could be removed
- In the 2nd example, we should pass `8` instead of `9` as segment count. The reason is that the explicitly given interval start/end limit the item count to 8 instead of 9 (like in the 1st example). I guess that `9` leads to misaligned (unbalanced) buckets which is why the result sparkbar looks so different from the one in the 1st example.